### DISCUSSION TITLE
Trying GPT 3 Proposal

### DISCUSSION THREAD

**Discussion by tsykes** [dbeaver/dbeaver#18994] [General]
I love the new AI extension for GPT. My first attempt was to get it to generate a template for Postgres that I'm always look up the syntax in the documentation every time I write it. 

Prompt: "Write a postgresql do block to iterate through a result set" 

When I ask this in the regular UI, it gives me a perfect response. So far, with the API settings, it's quite disappointing. 

I've tried both the code and text davinci models. The text seems to do a little better with at least giving me the start of a do block but it's not well formed. 

How are others using this? Any idea how to get closest to the code block in https://chat.openai.com/ would return?

**Comment by PUTHIYATH** [comment]
chatgpt uses this under the hood
https://platform.openai.com/playground
